__tests__/app.test.js

```diff
@@ -278,3 +278,57 @@ describe("task 7. POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("Task 8 /api/articles/:article_id, update an article by id", () => {
+  test("PATCH 201, should inc vote property by 1", () => {
```

Comment:
this should return a 200 ok status as we aren't creating a new resource so we wouldn't use 201